src/plugins/intel_gpu/tests/unit/fusions/reduce_fusion_test.cpp

```diff
@@ -67,6 +71,10 @@ class ReduceFusingTest : public ::BaseFusingTest<reduce_test_params> {
         return layout{ p.in_shape, p.data_type, p.input_format };
     }
 
+    layout get_default_format_input_layout(reduce_test_params& p) {
+        return layout{ p.in_shape, p.data_type, format::bfyx };
```

Comment:
to make it future proof, I think we should use `get_default_format(p.input_format)`

Reply:
I added a flag to get and to use default format in get_input_layout. How's it?

Reply:
After short discussion about 'default format', here it uses default formats given by format::get_default_format.